pipelines/pro_image.go

```diff
@@ -0,0 +1,75 @@
+package pipelines
+
+import (
+	"context"
+	"fmt"
+
+	"dagger.io/dagger"
+	"github.com/grafana/grafana-build/containers"
+)
+
+func ProImage(ctx context.Context, dc *dagger.Client, directory *dagger.Directory, args PipelineArgs) error {
+	debianPackageFile := dc.Host().Directory("./").File(args.ProImageOpts.Deb)
```

Comment:
We have a function to fetch this type of thing here:

https://github.com/grafana/grafana-build/blob/af3de95c7d5af7d7a988cb458c4068c53203b0b3/containers/package_input.go#L26

When you use this function, you give it a `file://path/to/file` argument or `gs://path/to/file.deb` to get a file from a GCS bucket.

It accepts a list of packages which isn't applicable in this case so we should extract the loop into a smaller function. Then you can get rid of the `--deb` argument and replace it with the `PackageInputFlags`.

Hope that makes sense. Basically I'm trying to make this look like the other pipelines that accept package(s) as input and produce something with them (like `cdn`, `rpm`, `deb`, `windows-installer`).

Reply:
Started using `containers.GetPackages` to get the package that should be built.